### ISSUE TITLE
last CDN's versions do not work

### ISSUE THREAD

**Issue by pavelnikitiuk** [prismicio/prismic-dom#40]
Yesterday I faced with a problem of this CDN script https://unpkg.com/prismic-dom 
it started to redirect to unstable versions of prismic-dom
https://unpkg.com/prismic-dom@2.1.1/dist/prismic-dom.min.js have an error when script loaded
```
Uncaught TypeError: Cannot assign to read only property 'exports' of object '#<Object>'
    at Module.<anonymous> (prismic-dom.min.js:1)
    at Module.<anonymous> (prismic-dom.min.js:1)
    at n (prismic-dom.min.js:1)
    at Object.<anonymous> (prismic-dom.min.js:1)
    at n (prismic-dom.min.js:1)
    at prismic-dom.min.js:1
    at Object.<anonymous> (prismic-dom.min.js:1)
    at n (prismic-dom.min.js:1)
    at Object.<anonymous> (prismic-dom.min.js:1)
    at n (prismic-dom.min.js:1)
```
current version 
https://unpkg.com/prismic-dom@2.2.1/dist/prismic-dom.min.js - have an error when script loaded
```
Uncaught ReferenceError: global is not defined
    at prismic-dom.min.js:1
```

**Comment by arnaudlewis** [comment]
Hello, a change in the build created this issue and we are actively working on it.
The issue is fixed for most cases but not yet for unpkg. I suggest you to fix your version to 2.1.0 and we'll keep you posted when the new version is available.

**Comment by arnaudlewis** [comment]
Sorry for the inconvenience, It's now fixed and you can fetch the last version `v2.2.3`.
Thank you for your patience.
